constraints.pro

```diff
@@ -144,16 +144,16 @@ gen_enforced_field(WorkspaceCwd, 'keywords', null) :-
 gen_enforced_field(WorkspaceCwd, 'homepage', CorrectHomepageUrl) :-
   \+ workspace_field(WorkspaceCwd, 'private', true),
   workspace_basename(WorkspaceCwd, WorkspaceBasename),
-  atomic_list_concat(['https://github.com/MetaMask/controllers/tree/main/packages/', WorkspaceBasename, '#readme'], CorrectHomepageUrl).
```

Comment:
See https://github.com/MetaMask/controllers/pull/1032. This PR makes it so that when we update the repository URL in the root `package.json`, all repo references in this file and other `package.json`s should be checked against that. Doesn't address the references in other files of course, but at least it's something.

Reply:
Thanks! Now that that PR has been merged, no changes are required in the constraints.